MITLibraries/alma-scripts
# Alma concat scripts

#### Developer Checklist

- [X] Code has been tested in stage
- [X] Documentation is available 

#### What does this PR do?

This is a first stab at a Full update concatenation script used for timdex and aleph.  The Full update script is the only one being used at the moment, but the Update script is included in case we need to start using it quickly.  

#### Helpful background context

This script touches parts of the changes to the terraform infrastructure to include environment variables in the ec2 instance to reference the aleph and timdex s3 buckets directly, and we use awscli to copy to and from those buckets.  

#### How to test
You could test this on stage by deleting the current full update file in the aleph bucket in stage, then re-running the script on alma-sftp-stage(making sure you're on this branch and up to date there) but we have tested the full-update script in stage successfully by running it on July's updated files already.  If you do this, you may want to coordinate with Helen to make sure its OK to do on stage.    

#### What are the relevant tickets?

https://mitlibraries.atlassian.net/browse/IMP-2130
